### DISCUSSION TITLE
Support for fmp4 (fragmented playable mp4 chunks)

### DISCUSSION THREAD

**Discussion by neilkinnish** [WebAV-Tech/WebAV#286] [General]
I can see that AV Recorder works with a time slice, I need to look deeper, but do you have support for chunking fmp4?

**Comment by hughfenghen** [comment]
@neilkinnish  The current output video file stream is fMP4.

**Comment by neilkinnish** [comment]
Hi, thanks. Can it output playable chunks, fragmented... so fmp4 chunks, as you record. (for example usable via HLS)

**Reply by hughfenghen** [reply]
It’s not supported by default, but it might be achievable with mp4box.js.
